mlir/lib/Dialect/SparseTensor/Transforms/SparseTensorConversion.cpp

```diff
@@ -271,62 +284,6 @@ class NewCallParams final {
   Value params[kNumParams];
 };
 
-// TODO: see the note at `_mlir_ciface_newSparseTensor` about how
```

Comment:
This is the essential part of the change. This whole block of code is replaced with the few lines above.